K2Bridge/Factories/BucketFactory.cs

```diff
@@ -181,8 +180,13 @@ public static void CreateAggregationColumns(Bucket bucket, DataRow row, ILogger
                     var columnName = column.ColumnName;
                     logger.LogTrace("Defining the value for {columnName}", columnName);
 
+                    if (!double.TryParse(row[columnName]?.ToString() ?? string.Empty, NumberStyles.Number, CultureInfo.InvariantCulture,  out var value))
```

Comment:
can we simplify this line ? ( ! ? ?? out)

Reply:
Is this better?